src/migtd/src/bin/migtd/main.rs

```diff
@@ -69,11 +69,11 @@ pub fn runtime_main() {
     handle_pre_mig();
 }
 
-pub fn basic_info() {
+fn basic_info() {
     info!("MigTD Version - {}\n", MIGTD_VERSION);
 }
 
-pub fn do_measurements() {
+fn do_measurements() {
```

Comment:
This function invokes mig_policy::init_tcb_info(), which will generate MigTD Quote and verify it. The PR pipeline emulation integration test runs on non-TDX VM, with `test_disable_ra_and_accept_all` flag to use a template REPORT/QUOTE in AzCVMEmu mode, which will not pass Quote verification. Need to bypass Quote verification in AzCVMEmu mode, with `test_disable_ra_and_accept_all` flag. Otherwise, pipeline test emulation integration with v2-policy enabled would fail always.